### PR TITLE
Fix storageProfile access_modes property

### PR DESCRIPTION
**Pull request by hmeir**
Right now in StorageProfile class, the function **first_claim_property_set_access_modes** getting the "volumeMode" instead of "accessModes".
So this PR is fixing it.

### CODE REVIEW

**Review by myakove — APPROVED** [review]

**Comment by redhat-qe-bot1** [comment]
Cherry-picked PR Fix storageProfile access_modes property into v4.15

File: ocp_resources/storage_profile.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def claim_property_sets(self):
         return self.instance.status.get("claimPropertySets")
 
     def first_claim_property_set_access_modes(self):
-        return self.claim_property_sets[0].get("volumeMode") if self.claim_property_sets else None
+        return self.claim_property_sets[0].get("accessModes") if self.claim_property_sets else None
```
Comment (jpeimer) [inline ocp_resources/storage_profile.py:21]: If we want to get the **first** access mode, shouldn't it be
`claim_property_sets[0].get("accessModes")[0]`
?

**Pull request merged**